manifests/init.pp

```diff
@@ -336,6 +336,48 @@
   Boolean $purge_plugins                          = $jenkins::params::purge_plugins,
 ) inherits jenkins::params {
 
+  validate_string($version)
```

Comment:
We use data types and since you're not changing anything, I think it should already be covered. Please have a look at the parameters to verify.

Reply:
You are correct. I removed all the validate statements since using types makes them moot